adoc/deployment-sysreqs.adoc

```diff
@@ -418,6 +418,9 @@ In the current state, full administrative access is granted to the cluster.
 Authentication is done via the kubeconfig file generated during deployment. This file will grant full access to the cluster and all workloads.
 Apply best practices for access control to workstations configured to administer the {productname} cluster.
 
+The {productname} needs an external authentication server like LDAP, AD, etc to validate the user's entity and grant different user roles or cluster role
```

Comment:
Suggest:
`The {productname} leverages {kube} role-based access control (RBAC) for authentication and will need to have an external authentication server such as LDAP, AD, etc to validate the user's entity and grant different user roles or cluster role permission..`